projects/igniteui-angular/src/lib/grids/common/grid.interface.ts

```diff
@@ -205,21 +205,23 @@ export interface ColumnType {
     populateVisibleIndexes?(): void;
 }
 
-export interface IFieldValid {
-    field: string;
-    valid: boolean;
-    formGroup: FormGroup;
+export interface IGridFormGroupCreatedEventArgs {
+    formGroup: FormGroup,
+    owner: GridType
 }
 
-export enum Validity {
-Valid,
-Invalid
+export interface IGridValidityStatusEventArgs {
+    status: ValidityStatus,
+    owner: GridType
 }
 
+export declare type ValidityStatus = 'VALID' | 'INVALID';
```

Comment:
Since this is not a definition and I don't thing we need it to be ambient, just the export will suffice:
```suggestion
export type ValidityStatus = 'VALID' | 'INVALID';
```